src/Extensions/Hosting.Services/IListenerBuilder.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Fabric;
+using Microsoft.ServiceFabric.Services.Communication.Runtime;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services
+{
+	/// <summary>Creates comunication listener for SF service</summary>
```

Comment:
Format this with `<summary>` on their own lines with a `<typeparam>` comment

Reply:
I've fixed summary, but I don't think that typeparam will add clarity here